proposals/evaluation/uc-3-inheritance.md

```diff
@@ -64,8 +63,18 @@ The access control `</weekly-status/.acp#ac1>` applies to all resources containe
   acp:applyMembers </acp/research#p1> . # applies the policy to all resources contained by </weekly-status/>
 ```
 
-Todo:  what is the ACR for 
-`</weekly-status/2021-04-28/report.md>` and what information is in it?
+An unauthenticated client making a GET on `</weekly-status/>` container will receive a `Link: <.acp>; rel="acl"` header in the `401` response that points to the above `<.acp>`. 
+This relation is what makes the contents of `</weekly-status/.acp>` authoritative, and is therefore the information the client can use to decide what credentials to present.
```

Comment:
I think this assumption can be made only in very limited number of real world scenarios. I don't see why we would want to make this assumption for this use case.

Reply:
What assumption do you have a problem with @elf-pavlik? 
I am building my server this way and that is how I built clients in the past.